service/ai.py

```
@@ -0,0 +1,18 @@
+
+load_dotenv()
+API_KEY = os.getenv("GOOGLE_API_KEY")
+genai.configure(api_key=API_KEY)
+
+MODEL_NAME = "gemini-2.5-flash"
+
+async def analyze_sentence(text: str) -> str:
+    try:
+        model = genai.GenerativeModel(MODEL_NAME)
+        response = model.generate_content(text)
+        return response.text
```

Ruff (0.14.7)

Consider moving this statement to an `else` block

(TRY300)